clustered/server/src/test/java/org/ehcache/clustered/server/store/ClusterTierActiveEntityTest.java

```diff
@@ -87,9 +87,9 @@
 import static org.hamcrest.Matchers.is;
 import static org.junit.Assert.assertThat;
 import static org.junit.Assert.fail;
-import static org.mockito.Matchers.any;
-import static org.mockito.Matchers.anyString;
-import static org.mockito.Matchers.eq;
+import static org.mockito.ArgumentMatchers.any;
```

Comment:
The any usage in `testReplicationMessageAndOriginalServerStoreOpMessageHasSameConcurrency()` should probably be `any(EntityMessage.class)` to ensure we fail if it's null.